services/frontend-v3/src/i18n/fr_CA.json

```
@@ -318,6 +318,10 @@
   "search.results.cards.connection.tooltip.hidden": "Cette personne est caché",
   "search.results.cards.connection.tooltip.inactive": "Cette personne est inactive et caché",
   "search.results.cards.remove.connection": "Déconnecter",
+  "search.results.cards.location.not.found": "Emplacement inconnu",
+  "search.results.cards.branch.not.found": "Branche inconnue",
+  "search.results.cards.skills.not.found": "Aucune compétence correspondante trouvée",
```

Since the english version is plural
```suggestion
  "search.results.cards.skills.not.found": "Aucunes compétences correspondantes trouvées",
```